core/amber/src/main/scala/edu/uci/ics/texera/web/model/websocket/event/BreakpointTriggeredEvent.scala

```diff
@@ -1,14 +1,8 @@
 package edu.uci.ics.texera.web.model.websocket.event
 
-import edu.uci.ics.texera.web.model.common.FaultedTupleFrontend
-
-case class BreakpointFault(
-    actorPath: String,
-    faultedTuple: FaultedTupleFrontend,
-    messages: Array[String]
-)
+import edu.uci.ics.texera.web.workflowruntimestate.BreakpointEvent
 
 case class BreakpointTriggeredEvent(
-    report: Array[BreakpointFault],
+    report: Iterable[BreakpointEvent],
```

Comment:
I don't think it is a good idea for an `Event` to contain another `Event`.

Reply:
renamed it back to `BreakpointFault`